snapcraft/models/project.py

```diff
@@ -737,8 +738,34 @@ def get_build_for_arch_triplet(self) -> Optional[str]:
 
     def get_build_plan(self) -> List[BuildInfo]:
         """Get the build plan for this project."""
-        # TODO
-        raise NotImplementedError("Not implemented yet!")
+        # pylint: disable=import-outside-toplevel
+        from snapcraft.providers import SNAPCRAFT_BASE_TO_PROVIDER_BASE
```

Comment:
Why not import at the top of the module?

Reply:
I guess it's a bit of a (bad) habit of mine, I'll revert this